server/routes/spotsHW.js

```diff
@@ -0,0 +1,28 @@
+var express = require('express');
+var uuid = require('uuid');
+var router = express.Router();
+
+var SPOTS_API_KEY;
+if(process.env.BUILD == 'development') {
+  SPOTS_API_KEY = '1234';
+} else {
+  SPOTS_API_KEY = uuid();
+}
+console.log("Using SPOTS_API_KEY = %s", SPOTS_API_KEY)
+
+router.post('/update/:key/:lotID/:spotID/:vacancy/:cardID', (req, res, next) => {
+  console.log(req.params);
+  key = req.params.key;
+  if(key != SPOTS_API_KEY) {
+    res.status(401).send({error:"invalid key"});
+    return;
+  }
+
+  if(req.params.cardID == '123456789') {
```

Comment:
@Anguian3, as you can see, this call will only work for one user. let me know if you want to do something different.